modules/k6/k6_test.go

```diff
@@ -0,0 +1,45 @@
+package k6
+
+import (
+	"bytes"
+	"context"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestK6(t *testing.T) {
+	ctx := context.Background()
+
+	absPath, err := filepath.Abs("./scripts/test.js")
```

Comment:
Let's build the path to be cross-OSs
```suggestion
	absPath, err := filepath.Abs(filepath.Join("scripts", "test.js"))
```